openshift/qontract-reconcile-internal.yaml

```diff
@@ -6961,6 +6778,63 @@ objects:
           emptyDir: {}
         - name: fluentd-config
           emptyDir: {}
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    labels:
+      app: qontract-reconcile-jenkins-job-builds-cleaner
+    name: qontract-reconcile-jenkins-job-builds-cleaner
+  spec:
+    schedule: "0 * * * *"
+    concurrencyPolicy: Allow
```

Comment:
Should we set this to `Forbid` or maybe `Replace` here? We wouldn't want multiple jobs to run at the same time